webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/GroupsApi.scala

```diff
@@ -353,7 +353,10 @@ class GroupApiService2 (
     implicit val prettify = restExtractor.extractPrettify(req.params)
     val modId = ModificationId(uuidGen.newUuid)
     val actor = RestUtils.getActor(req)
-    val groupId = NodeGroupId(req.param("id").getOrElse(uuidGen.newUuid))
```

Comment:
It could have been done like we do in Rules / Directives 

      val id = restExtractor.extractId(req)(x => Full(NodeGroupId(x))).map(_.getOrElse(NodeGroupId(uuidGen.newUuid)))